### ISSUE TITLE
Exception peek UI doesn't seem to function for me on version 1.18

### ISSUE THREAD

**Issue by StephenBloomquist** [Microsoft/vscode-debugadapter-node#110]
This is with VSCode version 1.11.1

In trying to move my debugger to version 1.18 of the protocol and adapter, but the exception UI (the red bar that spans the screen with exception details) no longer displays. My usage of stopped event hadn't changed between versions, and everything else about the exception behaves as expected.

I get these errors when I try resizing the debug side bar, which seem like they're relevant due to the exception widget calls in the stack trace:

![image](https://cloud.githubusercontent.com/assets/24838721/24772953/12c23dea-1ac8-11e7-8112-c53202611939.png)

**Comment by weinand** [comment]
This issue was moved to Microsoft/vscode#24411

**Comment by weinand** [comment]
This is a VS Code problem, not a debug adapter protocol issue.

**Comment by StephenBloomquist** [comment]
@weinand 
I had to go back and double check, since I forgot over the weekend, but the main reason why I posted this here is because the exception widget works for version 1.17 of the adapter, which is why I posted here. I couldn't be sure who was the cause.

**Comment by weinand** [comment]
@StephenBloomquist I've checked the changes between the 1.17 and 1.18 version of the adapter and I do not see any change that could explain why VS Code is crashing on the latter.

So what you are seeing is the following:
If you build your adapter with version 1.17 of the adapter library, VS Code 1.11.1 works fine and if you use the version 1.18 of the adapter library VS Code 1.11.1 will crash.

**Comment by StephenBloomquist** [comment]
Yes, but it doesn't really crash. The exception peek widget doesn't display, but everything behaves as expected otherwise _until_ I try resizing the window and that's when I get all those web errors.

**Comment by weinand** [comment]
I don't care whether we call this "crash" or not.

I only care that you can confirm that you have verified exactly the following:
- If you build your adapter with version 1.17 of the adapter library, VS Code 1.11.1 works fine
- if you build your adapter with version 1.18 of the adapter library VS Code 1.11.1 shows the problem.
- there are no other differences between the two setups

**Comment by weinand** [comment]
I did a bit compare between the 1.17 and 1.18 versions of vscode-debugadapter and the only changes are in the logger.ts/js (which you are probably not using anyway).

**Comment by StephenBloomquist** [comment]
Sorry, I was simply trying to be as specific as possible for you, wasn't intending to nitpick semantics.

Hmm. Okay I'm pretty sure it's not you guys now. I can't get it to work when building against 1.17 anymore (the usages of the exception and stopped event are unchanged). The older version I have which was built against 1.17 during VSCode 1.10 does still work. I'm guessing another package updated with 1.11.1 that I'm not taking into account here.

I'll try messing with some of the other package versions and get back to you.

**Comment by StephenBloomquist** [comment]
I can't for the life of me replicate this. I even copied all the exception-related code directly from the commit of the version that's working. 

I've tried building against all package version combinations for Adapter/Protocol versions 1.16, 1.17, and 1.18 with vscode package versions 1.0.5, 1.0.6, and 1.1.0 but behavior in all cases remains the bugged variant... I suspect I'm missing something but am unsure what it may be.

Right now the only build which doesn't have the bug is a .vsix package I built on April 5th
